.github/workflows/publish_docs.yml

```diff
@@ -5,7 +5,7 @@ on:
   workflow_dispatch:
 
 permissions:
-  deployments: write
+  contents: write
```

Comment:
Did we receive any notifications with the job failure? If not is there a way to ensure that for the future?

Reply:
An IM pointed out the docs seemed not up-to-date and then I noticed the [last 3 releases](https://github.com/Adyen/adyen-android/actions/workflows/finalize_release.yml) have the finalize release workflow failing. Let's discuss with the team how we missed it and how we could prevent that again 👍